packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/data/StudioIdentifiers.java

```diff
@@ -21,9 +21,13 @@
  */
 public final class StudioIdentifiers {
 
-    public static final String EMPTY_STUDIO_PROJECT = "250cabc0-a211-438c-8015-2d2aa136eb81";
+    public static final UUID EMPTY_STUDIO_PROJECT = UUID.fromString("bd3017e3-d95f-4535-8701-af6ba982619f");
 
-    public static final String SAMPLE_STUDIO_PROJECT = "01234836-0902-418a-900a-4c0afd20323e";
+    public static final UUID EMPTY_STUDIO_PROJECT_ID = UUID.fromString("250cabc0-a211-438c-8015-2d2aa136eb81");
```

Comment:
Why change everything to use `_ID`? Everything here is an identifier, what is the point of those two identifiers?

Reply:
I've explicitly declared an `_EDITING_CONTEXT_ID` to distinguish the two